components/shared/infoBox/InfoBox.tsx

```diff
@@ -19,39 +19,30 @@ const InfoBox = ({ work }: InfoBoxProps) => {
       animate={{ opacity: 1 }}
       transition={{ duration: 0.5, ease: "easeOut" }}>
       <section className="mt-14 w-full rounded-md bg-background-overlay px-6 pb-14 pt-8 lg:flex-row lg:px-14">
-        <h2 className="text-typo-heading-4 lg:mb-24">Beskrivelse</h2>
+        <h2 className="mb-10 text-typo-heading-4 lg:mb-24">Beskrivelse</h2>
         <div className="flex w-full flex-col gap-grid-gap-3 lg:flex-row lg:gap-44">
           <div className="flex-1 text-typo-body-md">
-            {!work.abstract ||
-              (!work.abstract.length && (
-                <p className="pt-5 lg:pt-0">Værket har desværre ingen beskrivelse.</p>
-              ))}
-            {work.abstract &&
-              work.abstract.length &&
-              work.abstract.map(abstract => (
-                <p key={abstract.substring(0, 30)} className="pt-5 lg:pt-0">
-                  {abstract}
-                </p>
-              ))}
+            {!work.abstract?.length && <p>Værket har desværre ingen beskrivelse.</p>}
+            {!!work.abstract?.length &&
+              work.abstract.map(abstract => <p key={abstract.substring(0, 30)}>{abstract}</p>)}
           </div>
           <dl className="flex-1">
-            {!!selectedManifestation?.audience && (
+            {!!selectedManifestation?.audience?.ages.length && (
               <InfoBoxItem
                 term="Alder"
-                description={selectedManifestation?.audience?.ages.map(age => age.display)}
+                description={selectedManifestation.audience.ages.map(age => age.display)}
               />
             )}
-            {!!selectedManifestation && (
+            {!!selectedManifestation?.series?.length && (
               <InfoBoxItem term="Serie" description={getSeriesInfo(selectedManifestation)} />
             )}
-            {!!selectedManifestation?.subjects.all &&
-              selectedManifestation?.subjects.all.length > 0 && (
-                <InfoBoxItem
-                  term="Emneord"
-                  description={selectedManifestation.subjects.all.map(subject => subject.display)}
-                  isButtons
-                />
-              )}
+            {!!selectedManifestation?.subjects.all?.length && (
+              <InfoBoxItem
+                term="Emneord"
+                description={selectedManifestation.subjects.all.map(subject => subject.display)}
+                isButtons
+              />
+            )}
```

Comment:
Have you considered if we want any fallback, if theese conditions are not met?

Reply:
Yesss, we agreed with them to not show the items then.